src/pages/Account.vue

```diff
@@ -12,9 +12,8 @@
                 <div v-if="voucherPayments.length > 0">
 
                     <div>
-                        <p>[xXx]Click the <span class="list-icon"><i class="fa fa-list" aria-hidden="true"></i></span> icon below to view a payment record in more detail.</p>
-                        <p>[xXx]To email yourself a specific payment record from the table below, select it and click 'Email selected payment record'.</p>
-                        <p>[xXx]Or, to email yourself all of your payment records, just click 'Email all payment records'.</p>
+                        <p>Click the <span class="list-icon"><i class="fa fa-list" aria-hidden="true"></i></span> icon below to view a payment record in more detail.</p>
+                        <p>To email yourself a specific payment record from the table below, select it and click 'Email selected payment record'.</p>
```

Comment:
Yep, I left this bit out intentionally because I think there's a lot of copy above the table and I think that (hopefully) the email all button doesn't actually need explanation.